gradle.properties

```diff
@@ -1,3 +1,3 @@
 org.gradle.daemon=true
-org.gradle.jvmargs=-Xmx2g -XX:+HeapDumpOnOutOfMemoryError -Xss2m
+org.gradle.jvmargs=-Xmx3g -XX:+HeapDumpOnOutOfMemoryError -Xss2m
```

Comment:
Were we seeing OOME before this change? If so, do we have an explanation why?

Reply:
Yes, this was the case when running `--parallel` without setting `max-workers` ( default was 16 on the machine ).  I have not looked further into it (yet) but have not ran into it with this setting. I assumed that it's because doing more things in parallel requires more memory.